### ISSUE TITLE
Add zuul.yaml

### ISSUE THREAD

**Issue by mrhillsman** [jitcoding/zuul-config-project#1]
Needed to work with Zuul

**Comment by mrhillsman** [comment]
Sample pipeline - check

```
- pipeline:
    name: check
    description: |
      Newly opened pull requests enter this pipeline to receive an
      initial verification
    manager: independent
    trigger:
      github:
        - event: pull_request
          action:
            - opened
            - changed
            - reopened
        - event: pull_request
          action: comment
          comment: (?i)^\s*recheck\s*$
    start:
      github:
        status: pending
        comment: false
    success:
      github:
        status: 'success'
    failure:
      github:
        status: 'failure'
```

**Comment by mrhillsman** [comment]
Fixed via 9600a52
